src/main/kotlin/com/openlattice/graph/PostgresGraphQueryService.kt

```diff
@@ -645,9 +645,20 @@ class PostgresGraphQueryService(
         return "SELECT * FROM ${EDGES.name} WHERE $idsClause $dstEntitySetIdsClause $associationEntitySetIdsClause $componentTypeClause"
     }
 
-    private fun inClause(uuids: Set<UUID>): String {
-        check(uuids.isNotEmpty()) { "Ids must be provided." }
-        return "IN (" + uuids.joinToString(",") { id -> "'$id'" } + ")"
+    private fun inClause(ids: Set<UUID>): String {
+        return "IN (" + ids.joinToString(",") { "'$it'" } + ")"
```

Comment:
why remove the non-empty check?

Reply:
oh nvm it's below

Reply:
inClause is still called from outside dataKeysClause though so it might be worthwhile to keep here

Reply:
?